models/WebsiteSetting.php

```diff
@@ -59,21 +64,29 @@ public static function getById(int $id): ?WebsiteSetting
     {
         $cacheKey = 'website_setting_' . $id;
 
-        try {
-            $setting = \Pimcore\Cache\RuntimeCache::get($cacheKey);
-            if (!$setting) {
-                throw new Exception('Website setting in registry is null');
-            }
-        } catch (Exception $e) {
+        if (\Pimcore\Cache\RuntimeCache::isRegistered($cacheKey)) {
             try {
```

Comment:
The try-catch block might not be necessary. I usually prefer to catch exceptions when defined, but we could consider removing it.

Reply:
Yeah I think so too. Since you already check if it is registered it should be fine without a try catch.
I guess if it would still throw an exception at this point, something else might have gone wrong. 

One minor thing maybe. Could you also import RuntimeCache as a whole instead of using \Pimcore\Cache\RuntimeCache?